lib/dotcom_web/components/route_symbols.ex

```diff
@@ -149,13 +165,46 @@ defmodule DotcomWeb.Components.RouteSymbols do
       type="icon-svg"
       name={"icon-logan-express-#{@route.name}"}
       class={"#{@class} #{@cva_class}"}
+      aria-label={route_label(@route)}
+      role="text"
     />
     """
   end
 
   def route_icon(assigns) do
     ~H"""
-    <.icon type="icon-svg" name="icon-mode-shuttle-default" class={"#{@class} #{@cva_class}"} />
+    <.icon
+      type="icon-svg"
+      name="icon-mode-shuttle-default"
+      class={"#{@class} #{@cva_class}"}
+      aria-label="Shuttle"
+      role="text"
+    />
     """
   end
+
+  # Given a route, return a machine-readable label.
+  defp route_label(%Route{description: :ferry}), do: "Ferry"
+  defp route_label(%Route{description: :regional_rail}), do: "Commuter Rail"
```

Comment:
I feel like the corresponding types (`2`, `4`) would be more reliable than these descriptions 🤷🏼‍♀️